apps/web/src/components/LiquidityCardRow/index.tsx

```diff
@@ -1,17 +1,20 @@
 import { Text, Card, Flex, Tag, SyncAltIcon } from '@pancakeswap/uikit'
 import NextLink from 'next/link'
 import { styled } from 'styled-components'
+import { useMemo } from 'react'
+import { Pool } from '@pancakeswap/v3-sdk'
 
 import DoubleCurrencyLogo from 'components/Logo/DoubleLogo'
 import { Percent, Currency } from '@pancakeswap/sdk'
+import { MerklRewardsTag } from 'views/Merkl/components/MerklTag'
```

Comment:
do you think it's better if we put this under components, as there's no actual views for merkl? Maybe put under `src/components/merkl/MerklTag`?

Reply:
for the hook how about just move it to `src/hooks`?

Reply:
Your suggestions's benefits:
1. enforce the global place for hooks and components. But it will cost higher context-switching when updating the feature.
2. maintain consistency 

putting all Merkl logic ( domain-focused ) in one place has 2 benefits:
1. context-switching is lower when searching and updating code
2. remove the feature faster

It depends on the team's choice. I will refer the domain-focused approach.

Reply:
I would vote for domain-focused structure if there's no potential dependency loop issue. 

The dependency flow should be `src/views` depends on `src/components`, not the other way around. And also views shouldn't rely on modules from other views as well, otherwise it'll be too hard to sort out the dependency relationship between each domain. In a word should be like this:

`src/pages` -> `src/views` -> `src/components`

But if we import `src/views` inside `src/components`, will violate the rule above.

Additionally, here're some of the rules I think we should follow:
:white_check_mark: `src/views/a` -> `src/views/a/hooks`
:white_check_mark: `src/views/a` -> `src/components`
:white_check_mark: `src/views/a` -> `src/hooks`
:white_check_mark: `src/components` -> `src/hooks`
:x: `src/views/a` -> `src/views/b`
:x: `src/components` -> `src/views/a`
:x: `src/hooks` -> `src/views/a`
:x: `src/hooks` -> `src/components`

Reply:
If we're following the domain-focused structure, I think we should put `LiquidityCardRow` under `views/liquidity`, and put `MerklTag` under either:
- `src/components` if it's a common component that can be used outside the domain of liquidity
- `views/liquidity` if it's solely designed to be placed inside the liquidity page, which means belongs to the domain of liquidity

while for the MerklSection, I believe it'll only be used inside the liquidity page, so it should be placed under the domain of liquidity

Reply:
It makes sense. I will update the change